Formula/switch-lan-play.rb

```diff
@@ -0,0 +1,19 @@
+class SwitchLanPlay < Formula
+  desc "Make you and your friends play games like in a LAN"
+  homepage "https://github.com/spacemeowx2/switch-lan-play"
+  url "https://github.com/spacemeowx2/switch-lan-play/archive/v0.0.7.tar.gz"
+  sha256 "0b645fa8bbff18b4b2d1dfd8d975790c6597bc0520c0ab1368387d08f8c2c900"
+
+  depends_on "cmake" => :build
+  depends_on "libuv"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/lan-play --version")
+    assert_match "1.", shell_output("#{bin}/lan-play --list-if")
```

Comment:
Test is a bit weak… is there more we could test?

Reply:
That's all we can test without external device.

Reply:
Could we spin up a server (docker is supported) to try and connect to that or is that overdoing it ?

Reply:
I think it can be tested with a server that doesn't exist(127.0.0.1:11451, `switch-lan-play` simply sends UDP packets to it). But I can't run `brew test` as root while `switch-lan-play` needs to be run as root(to use `libpcap`).

Reply:
So if it has to run as root I don't see we can do it. Does anyone else have any suggestions ?